opencti-platform/opencti-graphql/src/domain/settings.js

```diff
@@ -165,6 +163,10 @@ const ACCESS_SETTINGS_RESTRICTED_KEYS = [
   'password_policy_min_words',
   'password_policy_min_lowercase',
   'password_policy_min_uppercase',
+  'password_policy_min_uppercase',
+  'password_policy_min_uppercase',
```

Comment:
@JeremyCloarec little copy/paste error
```suggestion
  'password_policy_min_lowercase',
  'password_policy_min_uppercase',
```